### PR TITLE
Schnebly/optimize

### PR DESCRIPTION
**Pull request by schnebly**
This PR brings the time to calculate our 6 financial calculations (aka no II) to about 1.2-1.3 seconds. This is the V1 optimization as there is still one more optimization I would like to implement that could get us under 1 second per symbol but when I attempted it, I ran into some obstacles. It is definitely possible but it is less of a priority than coding up the Investability Index (II). Optimizations V2 will come after the II PR and scheduling PR.

### CODE REVIEW

File: analyzer-service/calculations.py [hunk 1]
```diff
@@ -4,20 +4,35 @@
 import statsmodels.api as sm
 from datetime import datetime
 
+import time
```
Comment (ayovev) [inline analyzer-service/calculations.py]: Once we feel good about the stability of our calculations process we might think about removing these timing statements but for right now I like using them as a good way of profiling/benchmarking our code

File: analyzer-service/calculations.py [hunk 2]
```diff
@@ -34,15 +49,12 @@ def loadHistorical(symbol):
 
   return data
 
+def getPctReturns(symbolData):
```
Comment (ayovev) [inline analyzer-service/calculations.py]: Good idea writing this utility function

File: analyzer-service/calculations.py [hunk 3]
```diff
@@ -57,86 +69,44 @@ def calculateValueAtRisk(symbolData):
...
   bReturns = bReturns[1:]
 
   model = sm.OLS(pReturns, bReturns).fit()
 
-  return {"R-Squared: {}".format(round(model.rsquared, 4))}
+  return round(model.rsquared, 4)
 
 
-def calculateExpectedReturn(symbolData, indexData):
+def calculateExpectedReturn(indexData, symbolBeta):
```
Comment (ayovev) [inline analyzer-service/calculations.py]: Yes 🌲

File: analyzer-service/calculations.py (hunk 1 above)
Comment (schnebly) [reply analyzer-service/calculations.py]: Yep, I'm on board.

**Pull request merged**